app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -12,8 +12,6 @@ class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
     require_nested :MiddlewareDatasource
     require_nested :MiddlewareMessaging
     require_nested :MiddlewareServer
-    require_nested :RefreshParser
-    require_nested :RefreshWorker
```

Comment:
Hmmm... This removal is wrong. May be I removed it by accident along with RefreshParser. RefreshWorker should still be there. Will re-add it.